shpkpr/resources/templates/marathon/default/standard.json.tmpl

```diff
@@ -0,0 +1,264 @@
+
+        When choosing a value for `cpus` you should take care not to request
+        more resources than your application needs:
+
+        - Is my application typically CPU bound?
+          - No: `0.1` is a good default value for `cpus`.
+          - Yes: `0.5` is a good starting value from which the application can
+            be monitored and adjusted up/down as necessary.
+    #}
+    "cpus": {{MARATHON_CPUS|default(0.1)|require_float(min=0.1)}},
+    {#
+        The amount of memory in MB that is needed for the application per
+        instance. If an application exceeds the amount of memory it has been
+        allocated, Marathon will kill and restart the task to reclaim the RAM.
+    #}
+    "mem": {{MARATHON_MEM|default(128)|require_int(min=0)}},
+    {# The number of instances of this application to start. #}
+    "instances": {{MARATHON_INSTANCES|default(1)|require_int(min=0)}},
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            {# The name/tag of the Docker image to use. #}
+            "image": "{{DOCKER_REPOTAG}}",
+            {% block docker_port_mappings %}
+            {% if DOCKER_EXPOSED_PORT is defined %}
+            "portMappings": [
+                {
+                    {#
+                        Port number within the container that should be exposed
+                        to the host.
+                    #}
+                    "containerPort": {{DOCKER_EXPOSED_PORT|require_int}},
+                    {#
+                        Setting `hostPort` to `0` allows Marathon to dynamically
+                        allocate a port on the host to map.
+                    #}
+                    "hostPort": 0,
+                    "protocol": "tcp"
+                }
+            ],
+            {% endif %}
+            {% endblock %}
+            "network": "BRIDGE"
+        }
+    },
+    {#
+        Key value pairs that get added to the environment variables of each task
+        that is started by Marathon.
+    #}
+    "env": {
+        {% for key, value in _all_env|filter_items("ENV_", strip_prefix=True) %}
+            "{{key}}": "{{value}}"{% if not loop.last %},{% endif %}
+        {% endfor %}
+    },
+    {#
+        Attaching metadata to apps can be useful to expose additional
+        information to other services, so Marathon has the ability to place
+        labels on apps (for example, you could label apps “staging” and
+        “production” to mark services by their position in the pipeline).
+    #}
+    {% block labels %}
+    "labels": {
+        {% for key, value in _all_env|filter_items("LABEL_", strip_prefix=True) %}
+            "{{key}}": "{{value}}"{% if not loop.last %},{% endif %}
+        {% endfor %}
+    },
+    {% endblock %}
+    {#
+        Secrets allow Marathon to retrieve secret data from configured "secret
+        stores" at application startup time. This allows decoupling of secret
+        provision from the application's use of the secrets.
```

Comment:
maybe "This allows decoupling of provisioning and storing secrets from the application's use of them."?